src/components/Restoration/RestorationType.vue

```diff
@@ -0,0 +1,136 @@
+              <div class="ml-8">
+                Please select
+                <v-tooltip content-class="top-tooltip" transition="fade-transition" top>
+                  <template v-slot:activator="{ on, attrs }">
+                    <span class="dotted-underline" v-bind="attrs" v-on="on">
+                      applicant's relationship
+                    </span>
+                  </template>
+                  Full restoration application must be related to the dissolved business.
+                </v-tooltip>
+                to the company at the time the company was dissolved:
+              </div>
+              <RelationshipsPanel :draftRelationships="getRestoration.relationships"
+                @changed="setRestorationRelationships($event)" @valid="setRestorationTypeValid($event)" />
+            </v-card>
+          </v-expand-transition>
+          <v-radio id="limited-radio-button" label="Limited Restoration" :value=RestorationTypes.LIMITED />
+
+          <!-- Limited Restoration Radio Panel -->
+          <v-expand-transition>
+            <v-card flat v-if="isLimited">
+              <LimitedRestorationPanel :currentDate="getCurrentDate" :expiryDate="getRestoration.expiry"
+                @expiry="setRestorationExpiry($event)" />
+            </v-card>
+          </v-expand-transition>
+
+        </v-radio-group>
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Watch, Emit, Prop } from 'vue-property-decorator'
+import { DateMixin, CommonMixin } from '@/mixins'
+import { Getter, Action } from 'vuex-class'
+import { RestorationTypes, RouteNames } from '@/enums'
+import { RelationshipsPanel } from '@bcrs-shared-components/relationships-panel'
+import { LimitedRestorationPanel } from '@bcrs-shared-components/limited-restoration-panel'
+
+@Component({
+  mixins: [
+    DateMixin,
+    CommonMixin
+  ],
+  components: {
+    RelationshipsPanel,
+    LimitedRestorationPanel
+  }
+})
+export default class RestorationType extends Vue {
+  @Getter getRestoration!: RestorationStateIF
+  @Getter getCurrentDate!: string
+  @Getter isRestorationTypeValid!: boolean
+
+  @Action setRestorationType!: ActionBindingIF
+  @Action setRestorationExpiry!: ActionBindingIF
+  @Action setRestorationRelationships!: ActionBindingIF
+  @Action setRestorationTypeValid!: ActionBindingIF
+
+  // Local properties
+  protected isFull = false
+  protected isLimited = false
+  protected selectRestorationType = ''
+
+  readonly RestorationTypes = RestorationTypes
+
+  /**
+   * Called when component is mounted.
+   * Automatically check limited or full restoration when user continues a draft as per last selection.
+   */
+  mounted (): void {
+    if (this.getRestoration.type === RestorationTypes.LIMITED) {
+      this.selectRestorationType = RestorationTypes.LIMITED
+      this.isFull = !this.isLimited
+    } else {
+      this.selectRestorationType = RestorationTypes.FULL
+    }
```

Comment:
Instead of this code, did you try initializing `selectRestorationType` from the store value? I think the property initialization is done when this component is created (not mounted), but it may work.
```
protected selectRestorationType = this.getRestoration.type
```

Reply:
This won't work! I tried it out, `this.getRestoration.type` is returning undefined.

Reply:
It might return null since that's what the value in the store is initialized to.

Anyway, this means this component is created before the restoration is fetched -- not surprising since the fetch is async (network call). I was hoping maybe this component wasn't rendered right away.

Side note: what would be the the harm in initializing `selectRestorationType` as null?

Reply:
Yes makes sense. There's no harm, I could do that. This is because I'm setting it again based on what was previously selected in the `mounted` block.

Reply:
Actually, it should be initialized to null. Currently you are initializing it to an empty string, but it's not a string.

What I mean is: what if you remove this code from `mounted()` ? Is there any issue with that?

Reply:
Thing is, in the mounted block, I am auto-selecting either the Full Restoration or Limited Restoration radio buttons when a user loads their previous draft. Without this code, when a user loads a draft, both radio buttons are not selected.

![mounted](https://user-images.githubusercontent.com/122301442/217681405-e623b4e7-6dea-4cdf-a33e-0b364f4cb8f6.PNG)

Reply:
Doesn't the first radio group get set as soon as the draft restoration filing is loaded?

Reply:
Does it work correctly atm or does something need to be figured out?

Reply:
> Doesn't the first radio group get set as soon as the draft restoration filing is loaded?

Without the mounted block, the expansion panel would work, but the radio buttons wouldn't be selected. They would both be empty as in the screenshot above.